nuxt/index.js

```diff
@@ -1,39 +1,64 @@
 const { resolve } = require('path')
 
-function pickFirst(...args) {
-  for (const arg of args) {
-    if (arg !== undefined) {
-      return arg
+module.exports = function nuxtBootstrapVue(moduleOptions = {}) {
+  this.nuxt.hook('build:before', () => {
```

Comment:
pi0 has said that he likes that nuxt-modules only run when they need to, unless I miss something I think this module only needs to run during build

Reply:
Yeah, I think that is correct, as the config options are defined at built time (and should not change after build)